submissions/chabuChabu/javaScript/index.js

```diff
@@ -0,0 +1,25 @@
+var productList = [
+   {
+    productName: "Red Tennis Shoes",
+    imageUrl: "../html/images/product-02.jpg",
+    productPrice: "$200",
+   },
+   {
+    productName: "Black Office shoes",
+    imageUrl: "../html/images/product-03.jpg",
+    productPrice: "$350",
+   },
+   {
+    productName: "Red High-heeled shoes",
+    imageUrl: "../html/images/product-04.jpg",
+    productPrice: "$230",
+   }
+]
+for(i = 0;i < productList.length;i++){
```

Comment:
@ChabuChabu `i` is not declared a variable.

Reply:
Commet addressed